GDJS/GDJS/IDE/Exporter.cpp

```diff
@@ -45,30 +45,54 @@ Exporter::Exporter(gd::AbstractFileSystem &fileSystem, gd::String gdjsRoot_)
 Exporter::~Exporter() {}
 
 bool Exporter::ExportProjectForPixiPreview(
-    const PreviewExportOptions &options) {
+    const PreviewExportOptions &options,
+    std::map<gd::String, gd::String> &projectPropertiesFallback) {
   ExporterHelper helper(fs, gdjsRoot, codeOutputDir);
-  return helper.ExportProjectForPixiPreview(options);
+  return helper.ExportProjectForPixiPreview(options, projectPropertiesFallback);
 }
 
 bool Exporter::ExportWholePixiProject(
     gd::Project &project,
     gd::String exportDir,
-    std::map<gd::String, bool> &exportOptions) {
+    std::map<gd::String, bool> &exportOptions,
+    std::map<gd::String, gd::String> &projectPropertiesFallback) {
   ExporterHelper helper(fs, gdjsRoot, codeOutputDir);
   gd::Project exportedProject = project;
 
   auto usedExtensionsResult = gd::UsedExtensionsFinder::ScanProject(project);
-  auto& usedExtensions = usedExtensionsResult.GetUsedExtensions();
+  auto &usedExtensions = usedExtensionsResult.GetUsedExtensions();
 
-  auto exportProject = [this, &exportedProject, &exportOptions, &helper,
+  auto exportProject = [this,
+                        &exportedProject,
+                        &exportOptions,
+                        &projectPropertiesFallback,
+                        &helper,
                         &usedExtensionsResult](gd::String exportDir) {
     bool exportForCordova = exportOptions["exportForCordova"];
     bool exportForFacebookInstantGames =
         exportOptions["exportForFacebookInstantGames"];
 
-    // Always disable the splash for Facebook Instant Games
-    if (exportForFacebookInstantGames)
+    // Always disable GDevelop branding for Facebook Instant Games
```

Comment:
I think we can remove this, it was a "gift" for early user but I don't think it's useful anymore